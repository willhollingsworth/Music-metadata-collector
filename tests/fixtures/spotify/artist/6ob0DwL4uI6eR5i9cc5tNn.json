{
  "external_urls": {
    "spotify": "https://open.spotify.com/artist/6ob0DwL4uI6eR5i9cc5tNn"
  },
  "followers": {
    "href": null,
    "total": 11230
  },
  "genres": [
    "melodic techno",
    "melodic house",
    "techno"
  ],
  "href": "https://api.spotify.com/v1/artists/6ob0DwL4uI6eR5i9cc5tNn",
  "id": "6ob0DwL4uI6eR5i9cc5tNn",
  "images": [
    {
      "url": "https://i.scdn.co/image/ab6761610000e5eb63afcfec0d4b39be6f44e50e",
      "height": 640,
      "width": 640
    },
    {
      "url": "https://i.scdn.co/image/ab6761610000517463afcfec0d4b39be6f44e50e",
      "height": 320,
      "width": 320
    },
    {
      "url": "https://i.scdn.co/image/ab6761610000f17863afcfec0d4b39be6f44e50e",
      "height": 160,
      "width": 160
    }
  ],
  "name": "widerberg",
  "popularity": 44,
  "type": "artist",
  "uri": "spotify:artist:6ob0DwL4uI6eR5i9cc5tNn"
}