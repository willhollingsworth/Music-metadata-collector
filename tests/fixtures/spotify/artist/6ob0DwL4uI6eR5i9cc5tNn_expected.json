{
  "artist_name": "widerberg",
  "artist_id": "6ob0DwL4uI6eR5i9cc5tNn",
  "followers": 11243,
  "popularity": 44,
  "albums": []
}